native/effektio/Cargo.toml

```diff
@@ -51,6 +52,9 @@ features = [
     "anyhow",
 ]
 
+[dependencies.ruma-common]
+features = ["unstable-msc3552"]
```

Comment:
I appreciate you finding out there is an open msc... but there's also an already implemented version we should be doing instead...

Reply:
Noted. Will check it.

Reply:
Just pushed #0eb3936393570afddc6ba244aaa9ab4ba7809277 (Replace msc3552 with standard on image send)